### DISCUSSION TITLE
Issues connecting to web brower

### DISCUSSION THREAD

**Discussion by abooosin** [Security-Onion-Solutions/securityonion#11246] [General]
![Screenshot (94).png](https://github.com/Security-Onion-Solutions/securityonion/assets/63638198/45f0d076-c751-4b11-aaaf-ba8816eb02b9)

Issues appeared when i tried to install onion on vmware and the SOC panel doesn't open at all

**Comment by dougburks** [comment]
Please provide more information as requested at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720.

Please include the following details where you can:

- Security Onion version as seen in the lower left corner of SOC and in /etc/soversion. For example: 2.4.10
- Is this a cloud deployment or on-prem? If on-prem, do you have Internet access or this an airgap installation?
- Did you install from our Security Onion ISO image or did you perform a network installation?
- If network installation, what distro and version did you install on?
- How many nodes do you have?
- What are the hardware specs of each of those nodes?
- How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)
- Are you experiencing issues monitoring network traffic? If so, are you sniffing from a tap or span port and what is the traffic volume?
- Does so-status show all services running?
- Do you get any failures when you run sudo salt-call state.highstate?
- Does the SOC Grid page show any failures?
- Explain your issue. For example: Installation fails when I select this series of options...
- Provide applicable logs. If you are having problems right after setup, provide /root/sosetup.log. If you are having problems during soup, provide /root/soup.log. If you are having problems with a specific component, provide that component's logs from /opt/so/log/.
